localrc-template

```diff
@@ -26,7 +26,7 @@ export SERVERS_PER_PORT=0  # 1 port per dev in ring; N obj-servers per port
 export REPLICATION_SERVERS=false  # or true
 export CONTAINER_AUTO_SHARD=true  # or false
 export OBJECT_SYNC_METHOD=rsync  # or ssync
-export USE_PYTHON3=true  # or false
+export USE_PYTHON3=false  # or false
```

Comment:
This is the opposite of what we should be doing in 2023 -- maybe we ought to make a separate `localrc-py2-template`, though? But see also: https://github.com/NVIDIA/vagrant-swift-all-in-one/pull/141

Reply:
see previous comment - oh, but I just realised that on github the 'Draft: ' prefix to the PR does not block merging :/